CONTRIBUTING.md

```diff
@@ -24,9 +24,9 @@ facilitates social contribution, easy testing and peer review.
 
 To contribute a patch, the workflow is as follows:
 
-  - Fork repository
-  - Create topic branch
-  - Commit patches
+  1. Fork repository
```

Comment:
Not sure what others think, but since it's markdown, all items can be ` 1. `.

Reply:
@promag Thanks for the suggestion! I just changed the same to suggested markdown format.